Doc/library/functions.rst

```diff
@@ -1593,8 +1593,8 @@ are always available.  They are listed here in alphabetical order.
 
 .. function:: super([type[, object-or-type]])
 
-   Return a proxy object that delegates method calls to a parent or sibling
-   class of *type*.  This is useful for accessing inherited methods that have
+   Return a proxy object that delegates attribute accesses to a parent or sibling
```

Comment:
I'd be inclined to just say "attribute access" instead of "attribute accesses".

Reply:
Done. Thanks @freddrake.